concepts/05 Widgets/Scheduler/040 Resources/030 Group Appointments by Resources.md

```diff
@@ -67,6 +67,88 @@ To group appointments by resources, assign an array to the [groups](/api-referen
         // ...
     })
 
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template>
+        <DxScheduler
+            :data-source="appointments"
+            :groups="groups" <!-- Groups appointments by rooms and by teachers -->
+        >
+            <DxResource
+                field-expr="roomId"
+                :data-source="roomResources"/>
```

Comment:
Let's add abbreviated data sources for rooms and teachers so that it doesn't look like we forgot them.

Reply:
Done